applications/asset_tracker_v2/src/main.c

```diff
@@ -543,7 +575,7 @@ void main(void)
 	int err;
 	struct app_msg_data msg;
 
-	if (!IS_ENABLED(CONFIG_LWM2M_CARRIER) && !IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)) {
+	if (!IS_ENABLED(CONFIG_LWM2M_CARRIER)) {
```

Comment:
I believe this will prevent nRF cloud fota from reporting the proper fota status.

Reply:
<img width="1261" alt="image" src="https://user-images.githubusercontent.com/4138391/171231762-478bd622-17dd-4c2d-8939-88d90350d8dc.png">

Reply:
since atv2 now reboots before nrf_cloud_fota is able to check the modem init return, nrf_cloud_fota is unable to validate that the update was installed successfully.  it does mark it as succeeded, with "not validated" in the status.

can you please re-add this check
`if (!IS_ENABLED(CONFIG_LWM2M_CARRIER) && !IS_ENABLED(CONFIG_NRF_CLOUD_FOTA)) {`